### ISSUE TITLE
AttributeError: 'module' object has no attribute 'NullHandler'

### ISSUE THREAD

**Issue by DavidAwad** [geduldig/TwitterAPI#53]
This is after importing straight out of a pip install. 
Macbook Air Newest OSX, python version 2.7.9. Not in a Virtualenv
```python
Traceback (most recent call last):
  File "app.py", line 2, in <module>
    from TwitterAPI import TwitterAPI
  File "/Library/Python/2.7/site-packages/TwitterAPI/__init__.py", line 12, in <module>
    logging.getLogger(__name__).addHandler(logging.NullHandler())
AttributeError: 'module' object has no attribute 'NullHandler'
```

**Comment by geduldig** [comment]
Please post some sample code that fails. I am having trouble reproducing this error. `NullHandler` was not included with Python until version 2.7, and Macs have several versions of Python pre-installed. Did you run `python --version` to confirm the version you are using?

Thanks,
Jonas

**Comment by aeeilllmrx** [comment]
I think I have this error as well.

[I 16:35:52.689 NotebookApp] Using MathJax from CDN: https://cdn.mathjax.org/mathjax/latest/MathJax.js
Traceback (most recent call last):
  File "/usr/local/bin/ipython", line 11, in <module>
...
...
    from .terminal import initialize
  File "/Library/Python/2.7/site-packages/IPython/html/terminal/__init__.py", line 3, in <module>
    import terminado
  File "/Library/Python/2.7/site-packages/terminado/__init__.py", line 7, in <module>
    logging.getLogger(__name__).addHandler(logging.NullHandler())
AttributeError: 'module' object has no attribute 'NullHandler'

alex$ python --version
Python 2.7.10
alex:python 

alex$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.10.1
BuildVersion:	14B25

Not sure if this is relevant, but this happened just after I downloaded and used a mysql database.
